src/coreclr/jit/lsraxarch.cpp

```diff
@@ -3136,4 +3305,78 @@ inline SingleTypeRegSet LinearScan::BuildEvexIncompatibleMask(GenTree* tree)
 #endif
 }
 
+//------------------------------------------------------------------------------
+// DoesThisUseGPR: Tries to determine if this node needs a GPR.
+//
+// Arguments:
+//    op   - the GenTree node to check
+//
+// Return Value:
+//    true if certain that GPR is necessary.
+//
+inline bool LinearScan::DoesThisUseGPR(GenTree* op)
+{
+    if (varTypeUsesIntReg(op->gtType))
+        return true;
```

Comment:
nit: add opening and closing curly braces.

Reply:
Changed